### ISSUE TITLE
How to set Full Texts display permanently as default in phpMyAdmin 4?

### ISSUE THREAD

**Issue by HakeraT** [phpmyadmin/phpmyadmin#11506]
As can be seen in the source code ( https://github.com/phpmyadmin/phpmyadmin/tree/master/libraries ) in PhpMyAdmin 4.*, "phpmyadmin/libraries/display_tbl.lib.php" no longer exists. So, it not possible to do  $_REQUEST['display_text'] = 'F'  the way it is described here: http://stackoverflow.com/a/17838640

If I change it through "Settings > Main Panel > Browse Mode: Limit column characters " the settings are only stored for the session, but I want it to be stored permanently.

How to set full text as default now?

**Comment by lem9** [comment]
For the current 4.5.0 version, I just added an answer to the stackoverflow question. Please upvote my answer.

**Comment by still-dreaming-1** [comment]
I'm confused. The StackOverflow accepted answer just talks about a "hacky" workaround, but the answer from @lem9 seems to mention an official patch. But I still cannot find any official way to set this default.

**Comment by ibennetch** [comment]
@still-dreaming-1 The accepted answer is not the one referred to by lem9. His answer is https://stackoverflow.com/a/32808952/2385479; which says:

> The equivalent patch for current stable version (4.5.0):

```
diff --git a/libraries/DisplayResults.class.php b/libraries/DisplayResults.class.php
index e435a21..7129c15 100644
--- a/libraries/DisplayResults.class.php
+++ b/libraries/DisplayResults.class.php
@@ -4101,7 +4101,9 @@ class PMA_DisplayResults
         } elseif (empty($query['pos'])) {
             $query['pos'] = 0;
         }
-
+        if (!isset($_REQUEST['pftext'])) {
+            $_REQUEST['pftext'] = self::DISPLAY_FULL_TEXT;
+        }
         if (PMA_isValid(
             $_REQUEST['pftext'],
             array(
```

**Comment by ibennetch** [comment]
If this is something that people want, we can look in to whether it makes sense to add this as a feature moving forward.

**Comment by still-dreaming-1** [comment]
Thanks, I have since discovered the $cfg['LimitChars'] configuration setting, which I then realized is what I really wanted. That way in case the text is extraordinarily long, I will not see all of it unless I explicitly do a one time full texts query. In other words this setting allows me to see the full texts of long texts by default without having to see extraordinarily long texts by default, and it allows me to define what I consider to be extraordinarily long.
